gradle/libs.versions.toml

```diff
@@ -43,7 +43,7 @@ apacheMaven-pluginTools = "3.5.2"
 kotlinx-cli = "0.3.6"
 
 ## NPM | Frontend
-node = "16.13.0"
+node = "18.20.4"
```

Comment:
As someone who doesn't know much about JS development, can you give a brief explanation why v18 should be chosen? Isn't v20 the latest stable lts release?

Reply:
Honestly there was no specific reasons, just a step-by-step update. I've tested it with v20 and it seems to work ok so I updated to v20